.github/workflows/test.yaml

```diff
@@ -210,15 +210,15 @@ jobs:
               "
           for EXPECTED_LINT_DEP in $EXPECTED_LINT_DEPS; do
               # Check that the `lint: ` includes the expected dependency
```

Comment:
Should we review the comment based on the chage ?

Reply:
oops, fixed, thanks